specifications/16-case-insensitive-in-operators.json

```diff
@@ -0,0 +1,163 @@
+{
+  "name": "16-case-insensitive-in-operators",
+  "state": {
+    "version": 2,
+    "features": [
+      {
+        "name": "F9.globalSegmentAndConstraint_IN",
```

Comment:
Last thing, the names don't make sense anymore now that we've removed the segments. If you patch the names here I'm happy to approve

Reply:
👍